README.md

```diff
@@ -1,26 +1,29 @@
 <p align="center">
-  <img src="assets/logos/logo.svg" alt="Meilisearch" width="200" height="200" />
+  <a href="https://www.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides" target="_blank">
+    <img src="assets/logos/logo.svg" alt="Meilisearch" width="200" height="200" />
+  </a>
 </p>
 
 
 <h1 align="center">Meilisearch Integrations</h1>
-<h5 align="center">This repository is the central reference for developing and maintaining integrations for <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a>.
+<h5 align="center">This repository is the central reference for developing and maintaining integrations for <a href="https://www.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides" target="_blank">Meilisearch</a>.
 </h5>
 
 <h4 align="center">
-  <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a> |
-  <a href="https://docs.meilisearch.com">Documentation</a> |
-  <a href="https://discord.meilisearch.com">Discord</a> |
-  <a href="https://www.meilisearch.com">Website</a> |
-  <a href="https://blog.meilisearch.com">Blog</a> |
-  <a href="https://fr.linkedin.com/company/meilisearch">LinkedIn</a> |
-  <a href="https://twitter.com/meilisearch">Twitter</a> |
-  <a href="https://www.meilisearch.com/docs/faq/">FAQ</a>
+  <a href="https://www.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">Website</a> |
+  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
+  <a href="https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">Meilisearch Cloud</a> |
+  <a href="https://blog.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">Blog</a> |
+  <a href="https://docs.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">Documentation</a> |
+  <a href="https://docs.meilisearch.com/faq/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">FAQ</a> |
+  <a href="https://discord.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=integration-guides">Discord</a>
 </h4>
 
-**The Integration Team owns all the repositories and tools that help the users integrate Meilisearch in their project.**
+> Meilisearch is an open-source search engine that offers fast, relevant search out of the box.
 
-## 🛠 Our Integrations
+👉 [Meilisearch repository](https://github.com/meilisearch/meilisearch)
```

Comment:
Why do you change this? I would rather we mention the purpose of the repo and the integration scope here. But indeed, since there is no official integration team, we can maybe remove the mention of "integration team" and talk about the integration scope 
In any case, does not seem really related to the PR -> we maybe could focus on changing the UTM in this PR firs 😊

Reply:
As you said, there is no integration team. Plus, this information is not relevant for most readers. They just want to learn how to use Meilisearch.

Happy to remove this from the PR but I don't see a reason why you would object to this change.

Reply:
I would like you to let the sentence talk about the integrations.
You can change for example, by 

"**The Meilisearch integrations are all the repositories and tools that help the users integrate Meilisearch in their project.**

Reply:
I was talking about the fact it's not related to the PR, because you made a change I don't fully agree, and since it's not related at all to the original change of the PR, it's too bad I have to prevent the merge because of this.

Reply:
Don't you think the h5 headline "This repository is the central reference for developing and maintaining integrations for Meilisearch" is explicit enough?

Moreover, the following section is named "Meilisearch Integrations" and starts by listing the integrations. The examples should clarify that enough IMO; and it's not like developers don't know what "integrations" means.

What do you think your proposal adds? Maybe I'm missing something. 😅

Reply:
> Don't you think the h5 headline "This repository is the central reference for developing and maintaining integrations for Meilisearch" is explicit enough?

It does not explain what integrations are.

> and it's not like developers don't know what "integrations" means.

I know from experience it's not always the case, developers have different definitions of what integrations are. For example, this is often something I have to explain to candidates during interviews when I detail the different scopes of Meiisearch.

To be honest, I felt free to ask for a [change](https://github.com/meilisearch/integration-guides/pull/293#discussion_r1380033642) since this repo is under my scope, but I did not initially plan to have this kind of long discussion for a small request of change like this 😅 so, even if I'm not 100% ok with the current state of the PR, I will merge it as it is because I think the most important is to fix UTM as a priority.